package.json

```diff
@@ -10,6 +10,7 @@
     "lint": "eslint src/**/*.{js,jsx,json} --fix"
   },
   "dependencies": {
+    "prop-types": "^15.8.1",
```

Comment:
Why did you install this?

Reply:
I tried to fix an error, but I see that doesn't appear without it now :P